hash/sega_beena_cart.xml

```diff
@@ -31,10 +84,103 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="btowneyo" supported="no">
+		<description>Beena Town e Youkoso</description>
+		<year>2005</year>
+		<publisher>Sega Toys</publisher>
+		<info name="alt_title" value="ビーナタウンへようこそ"/>
+		<part name="cart" interface="sega_beena_cart">
+			<dataarea name="rom" size="0x400000" width="32" endianness="big">
+				<rom loadflag="load16_word_swap" name="S-100001-1000 - Beena Town e Youkoso (Japan).bin" size="0x400000" crc="b86b8d11" sha1="7fae650ccfb574a3d460a5a7e42cad099d06e7e4" />
+			</dataarea>
+			<dataarea name="pages" size="0x2500000"> <!-- book pages -->
+				<rom name="0 - 0001.png" size="0x1ef651a" crc="79431205" sha1="3cb20f80018f356f711876e9f3623e0bb4fd8dba" />
+				<rom name="0 - 0002.png" size="0x1fde709" crc="72778f05" sha1="998f3c6bbd5bdabf1150fe5dcfa5eee0dfc85083" />
+				<rom name="0 - 0003.png" size="0x1fbfd28" crc="e4f1a9b4" sha1="e88da1ee32615d263b616fc495f5f271892b7d82" />
+				<rom name="0 - 0004.png" size="0x1ef0e79" crc="1945ee85" sha1="23f2db46075b1d5be48a0df680235b8dc39b62d2" />
+				<rom name="0 - 0005.png" size="0x1fbfd2b" crc="4fba7dfa" sha1="bf86eb4bb3f278f425c666b13bf7dd8ecb8ca557" />
+				<rom name="0 - 0006.png" size="0x1be6425" crc="5e439b1e" sha1="dc2e26a93644bd02b6fc4cb803509457863af333" />
+				<rom name="0 - 0007.png" size="0x1cd7f10" crc="140f5dca" sha1="76e5ebf5fa61fc7c0962cb222d9836df1be62d30" />
+				<rom name="0 - 0008.png" size="0x203de88" crc="f8b7cca2" sha1="62be480fe570c4a3724693379fe2917af21c428f" />
+			</dataarea>
+		</part>
+	</software>
+
+	<software name="dtanoshib" supported="no">
+		<description>Disney Tanoshii o-Benkyou Series: Mic de Kantan ABC</description>
```

Comment:
The honorific “o” is part of the word and should be capitalised if you’re writing in title case.

Reply:
Fixed, thanks!